modules/react/loading-dots/lib/LoadingDots.tsx

```diff
@@ -18,9 +18,10 @@ const keyframesLoading = keyframes({
 const singleLoadingDotStencil = createStencil({
```

Comment:
```suggestion
export const singleLoadingDotStencil = createStencil({
```

Reply:
If we export this, people can use the stencil to override styles as well!

Reply:
Does that work if we don't also expose the `LoadingAnimationDot` component, do overrides on a parent apply to children? (I'm really not familiar with this so please correct ruthlessly 😂 ) 

```tsx
// In consumer
const loadingDotsStyles = createStyles({
  [singleLoadingDotStencil.vars.color]: system.color.icon.inverse,
});

<LoadingDots cs={loadingDotsStyles}

// LoadingDots.tsx implementation
export const LoadingDots = createComponent('div')({
  displayName: 'LoadingDots',
  Component: (elemProps: CSProps, ref, Element) => {
  Component: ({backgroundColor, ...elemProps}: LoadingDotsProps, ref, Element) => {
    return (
      <Element ref={ref} {...handleCsProp(elemProps, loadingDotsStencil())}> // I was assuming `loadingDotsStyles` was being applied here
        <LoadingAnimationDot /> // Does this get the override in `loadingDotsStyles`?
        <LoadingAnimationDot />
        <LoadingAnimationDot />
      </Element>
    );
  },
});
```